x-pack/plugins/alerting/server/rules_client/methods/bulk_disable.ts

```diff
@@ -159,6 +159,7 @@ const bulkDisableRulesWithOCC = async (
         const updatedAttributes = updateMeta(context, {
           ...rule.attributes,
           enabled: false,
+          running: false,
```

Comment:
**question:** It's possible the rule is still running when disabling, should we keep the value unchanged here?